frontend/src/App.jsx

```diff
@@ -72,7 +72,7 @@ function App() {
   return (
     <div className="App">
       <Notifier />
-      {showModal && createPortal(<LogoutModal onClose={closeModal} onLogout={handleLogout} />, document.body)}
+      {showModal && createPortal(<ActionModal title="Logout" content={["Do you want to disconnect your wallet and logout of this account?"]} cancelLabel='Cancel' submitLabel="Yes, logout" submitAction={handleLogout} cancelAction={closeModal}/>, document.body)}
```

Comment:
this question text should be a subtitle, rather than a content

Also please format your all files, currently it is rendering almost in one line in such places, but should be split into next lines